examples/flow_around_cylinder_lowRe.jl

```diff
@@ -0,0 +1,119 @@
+# # Steady-state flow around a cylinder in 2D using immersed boundaries
```

Comment:
Since this immersed boundary implementation is experimental (and is not expected to be fully supported), does it make sense to put this script in `validation/immersed_boundaries/flow_around_cylinder.jl`?

I also suggest omitting "low Re", since "low Reynolds number" means different things to different people, unfortunately.

Reply:
Sounds good! That was mostly a hangover from naming convention in my own examples